src/sentry/static/sentry/app/views/organizationMonitors/monitorStats.jsx

```diff
@@ -0,0 +1,91 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import AsyncComponent from 'app/components/asyncComponent';
+import {Panel, PanelBody} from 'app/components/panels';
+import {t} from 'app/locale';
+import EmptyMessage from 'app/views/settings/components/emptyMessage';
+import StackedBarChart from 'app/components/stackedBarChart';
+
+export default class MonitorStats extends AsyncComponent {
+  static propTypes = {
+    monitor: PropTypes.object.isRequired,
+    ...AsyncComponent.PropTypes,
+  };
+
+  getDefaultState() {
+    const until = Math.floor(new Date().getTime() / 1000);
```

Comment:
Do these ever change?

Reply:
not currently, but this is how other stats was implemented with the assumption that they will be changed based on some other state (e.g. a quick selector to change time period of chart)